pkg/network/multus.go

```diff
@@ -107,3 +107,32 @@ func renderMultus(conf *cnao.NetworkAddonsConfigSpec, manifestDir string, opensh
 
 	return objs, nil
 }
+
+// RenderMultus generates the manifests of Multus
+func renderMultusV3(conf *cnao.NetworkAddonsConfigSpec, manifestDir string, openshiftNetworkConfig *osv1.Network, clusterInfo *ClusterInfo) ([]*unstructured.Unstructured, error) {
+	if conf.MultusV3 == nil || openshiftNetworkConfig != nil {
+		return nil, nil
+	}
+
+	// render manifests from disk
+	data := render.MakeRenderData()
+	data.Data["Namespace"] = os.Getenv("OPERAND_NAMESPACE")
+	data.Data["MultusImage"] = os.Getenv("MULTUS_V3_IMAGE")
+	data.Data["ImagePullPolicy"] = conf.ImagePullPolicy
+	data.Data["Placement"] = conf.PlacementConfiguration.Workloads
+	if clusterInfo.OpenShift4 {
+		data.Data["CNIConfigDir"] = cni.ConfigDirOpenShift4
+		data.Data["CNIBinDir"] = cni.BinDirOpenShift4
+	} else {
```

Comment:
I thought we wanted to prevent multus from being deployed in openshift.

Am I missing something ?

Reply:
I followed the example of Multus component we have today.

Reply:
we wont reach here due to 
https://github.com/kubevirt/cluster-network-addons-operator/pull/1513/files#diff-f905e8c1fce38a3ea9acd2b9fe20ab7b1217840a4e47c65595a9682491dbaf5bR113

right ?
so worth imo to remove for the new function in this PR